osx/src/main.cpp

```diff
@@ -3,7 +3,9 @@
 #include "data/clientGeoJsonSource.h"
 #include <cmath>
 #include <memory>
-
+#include <signal.h>
+#include <stdlib.h>
```

Comment:
".h" system headers are technically deprecated, prefer `<csignal>`, `<cstdlib>`, etc. in the future.